unit/java_bytecode/java_bytecode_parse_generics/parse_generic_wildcard_function.cpp

```diff
@@ -7,61 +7,64 @@
 \*******************************************************************/
 
 #include <testing-utils/catch.hpp>
+#include <testing-utils/load_java_class.h>
 
 #include <util/config.h>
-#include <util/cmdline.h>
 #include <util/language.h>
-#include <util/prefix.h>
 
 #include <java_bytecode/java_bytecode_language.h>
 
-#include <iostream>
-#include <testing-utils/load_java_class.h>
-
 SCENARIO(
-  "java_bytecode_parse_generic_wildcard",
+  "parse_generic_wildcard",
   "[core][java_bytecode][java_bytecode_parse_generics]")
 {
-  const symbol_tablet &new_symbol_table=
-    load_java_class(
-      "WildcardGenericFunctions",
-      "./java_bytecode/java_bytecode_parse_generics");
+  const symbol_tablet &new_symbol_table = load_java_class(
+    "WildcardGenericFunctions", "./java_bytecode/java_bytecode_parse_generics");
 
-  std::string class_prefix="java::WildcardGenericFunctions";
+  std::string class_prefix = "java::WildcardGenericFunctions";
 
   THEN("There should be a symbol for processSimpleGeneric")
   {
-    const std::string func_name=".processSimpleGeneric";
-    const std::string func_descriptor=":(LSimpleGeneric;)V";
-    const std::string process_func_name=class_prefix+func_name+func_descriptor;
+    const std::string func_name = ".processSimpleGeneric";
+    const std::string func_descriptor = ":(LGeneric;)V";
+    const std::string process_func_name =
+      class_prefix + func_name + func_descriptor;
 
     REQUIRE(new_symbol_table.has_symbol(process_func_name));
+    // TODO extend tests when wildacrds are parsed correctly - TG-675
```

Comment:
Minor nit (unfortunately copy-and-pasted a few times, so perhaps you can do a search-and-replace to fix) - `wildcards`